extract-jira-link-pages.sql

```diff
@@ -1,6 +1,22 @@
-SELECT c.contentid AS "Page ID", c.title AS "Page Title", c.version, o.text_val
-FROM CONTENT c, OS_PROPERTYENTRY o
-WHERE c.prevver is NULL 
-AND c.contentid=o.entity_id 
-AND o.entity_key=concat('~metadata.',c.version)
-AND o.text_val LIKE '%structured-macro ac:name=&quot;jira&quot;%'
\ No newline at end of file
+select
+	c.contentid as "Page ID",
+	c.title as "Page Title",
+	c."version",
+	op2.text_val
+from
+	os_propertyentry op2
+inner join (
+	select
+		distinct(entity_id),
+		max(entity_key) as "entity_key"
+	from
+		os_propertyentry op
+	where
+		op.entity_key like '~metadata.%'
+        and op.text_val like '%structured-macro ac:name=&quot;jira&quot;%'
+	group by
+		entity_id) op3 on
+	op2.entity_id = op3.entity_id
+	and op2.entity_key = op3.entity_key
+inner join "content" c on
+	c.contentid = op2.entity_id
```

Comment:
Are we telling the customer about `limit` and `offset`?

Reply:
Thanks for the reminder, I'll add that to the README.